packages/cli/src/task-list.ts

```diff
@@ -282,4 +284,15 @@ export default class TaskListImpl implements RegisterableTaskList {
   private addError(taskName: TaskName, error: Error) {
     this._errors.push({ taskName, error });
   }
+
+  private addErrors(taskName: TaskName, errors: Error[]) {
+    this._errors.push(
+      ...errors.map((error) => {
+        return {
+          error,
+          taskName,
+        };
+      })
+    );
+  }
```

Comment:
Let's change this to 

```suggestion
  private addErrors(taskName: TaskName, errors: Error | Error[]) {
    if (!Array.isArray(errors)) {
      errors = [errors];
    }
  
    this._errors.push(
      ...errors.map((error) => {
        return {
          error,
          taskName,
        };
      })
    );
  }
```

and then just delete the `addError` function.

Reply:
lol just did exactly that but no mutation and with a ternary so #oneline ;) updating